game/src/ecs/items/ItemData.java

```diff
@@ -0,0 +1,181 @@
+package ecs.items;
+
+import configuration.ItemConfig;
+import ecs.components.AnimationComponent;
+import ecs.components.HitboxComponent;
+import ecs.components.InventoryComponent;
+import ecs.components.ItemComponent;
+import ecs.components.PositionComponent;
+import ecs.components.stats.DamageModifier;
+import ecs.entities.Entity;
+import graphic.Animation;
+import java.util.List;
+import starter.Game;
+import tools.Point;
+
+public class ItemData {
```

Comment:
mal doof gefragt, könnte das ein `record` sein?

Reply:
ja, würde es aber lieber als Klasse erstmal lassen, da es erstmal weniger einschränkt in der Kreativität. Hier würde ich auch vermuten, dass die DSL es auch Schritt für Schritt anpassen würde wie sonst auch. Somit aus unserer aktuellen Sicht eher für record und Vermute auf Dauer wird klasse sinnvoller sein.

Reply:
Das mit der DSL ist ein gutes Argument.